README.md

```diff
@@ -52,6 +52,31 @@ If you don't want to enter `dev-env` manually each time using `eval "$(dev-env/b
 you can also install [direnv](https://direnv.net). This repo already provides a `.envrc`
 file, with an option to add more in a `.envrc.private` file.
 
+#### Mac
+
+On Mac `dev-env` can be installed with:
+
+1. Install Nix by running: `bash <(curl -sSfL https://nixos.org/nix/install)`
+   Note that Nix recommends a *multi-user installation* on macOS. You need to configure `/etc/nix/nix.conf` To use Nix caches.
+   You can add the contents of `dev-env/etc/nix.conf` to `/etc/nix/nix.conf`, but keep `build-users-group = nixbld` instead of leaving this empty as is done in `dev-env/etc/nix.conf`. Make sure to restart the `nix-daemon` after you have made changes to `/etc/nix/nix.conf`, for instance by using `sudo killall nix-daemon`.
```

Comment:
Doesn't MacOS have something along the lines of `systemd` to restart services? Is `sudo killall` safe?

Reply:
Yes, you could do 

```
sudo launchctl stop org.nixos.nix-daemon
sudo launchctl start org.nixos.nix-daemon

Reply:
done.